mozilla/blurts-server
# Remove empty section when env PREMIUM_ENABLED is not truthy

# Description

Removes an empty section when env PREMIUM_ENABLED is not truthy.

# Screenshot

| Before | After |
| --- | --- |
| <img width="965" alt="dashboard-empty-section-before" src="https://github.com/mozilla/blurts-server/assets/13835474/e2a9402e-6693-4c84-8542-dfa44293201e"> | <img width="959" alt="dashboard-empty-section-after" src="https://github.com/mozilla/blurts-server/assets/13835474/dd48800c-e90e-45a1-86ca-77ceb88dc647"> |

# How to test

Visit `/users/breaches` and make sure there is no empty space before the breaches card as seen on the screenshots.
